gensim/models/logentropy_model.py

```diff
@@ -3,17 +3,18 @@
 #
 # Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
 
+"""This module contains LogEntropyModel class"""
```

Comment:
Need to add more description "what is it"
- small description with links
- executable example